packages/components/app/styles/components/table.scss

```diff
@@ -16,32 +17,53 @@
 
 .hds-table--striped {
   tbody {
-    tr:nth-child(odd) {
+    tr:nth-child(even) {
       background-color: var(--token-color-surface-faint);
     }
   }
 }
 
+.hds-table--density-short td {
+  padding: 4px 16px;
+}
+
+
+.hds-table--density-medium td {
+  padding: 12px 16px;
+}
+
+.hds-table--density-tall td {
+  padding: 20px 16px;
+}
+
 .hds-table__thead {
   border-bottom: 1px solid var(--token-color-border-primary);
 
-  th {
+  .hds-table__th,
+  .hds-table__th-sort {
+    height: 48px;
     padding: 8px 16px;
     color: var(--token-color-foreground-strong);
     font-weight: var(--token-typography-font-weight-semibold);
+    font-size: 14px;
+    line-height: 1.4;
     text-align: left;
     background-color: var(--token-color-surface-strong);
+  }
 
-    &.hds-table__th-sort {
-      padding: 0; // we're taking it out here because we need it on the interactive content instead
-    }
+  .hds-table__th-sort {
+    padding: 0; // we're taking it out here because we need it on the interactive content instead
 
-    button { // will refactor this to a button. Use Hds::Button?
+    button {
       display: flex;
       align-items: center;
       width: 100%;
+      height: 100%;
       padding: 8px 16px;
       color: var(--token-color-foreground-strong);
+      font-weight: var(--token-typography-font-weight-semibold);
```

Comment:
I think I have already mentioned elsewhere, but here you can use `inherit` for the font stylings